BodoSQL/calcite_sql/bodosql-calcite-application/src/main/java/com/bodosql/calcite/rel/metadata/BodoRelMetadataProvider.kt

```diff
@@ -50,6 +51,10 @@ class BodoRelMetadataProvider(
                 BodoRelMdColumnDistinctCount(),
                 ColumnDistinctCount.Handler::class.java,
             ),
+            ReflectiveRelMetadataProvider.reflectiveSource(
```

Comment:
This is how you register new metadata. This makes it the first option (will fallback to Calcite for all non-Join implementations).